README.md

```diff
@@ -0,0 +1,161 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Budget App</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [Live Version](#live-version)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budget App <a name="about-project"></a>
+
+**Budget App** is a web app built with ruby on rails and uses postgresql database. It allowsa user to sign up, log in, create new groups and new records for their expenses(entities).
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://ruby.org/">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- **The page is built with Ruby on Rails**
+- **Sign up and Log in**
+**Add groups (categories) and expenses**
+
+### Live demo <a name="live-demo">
+[Budget App](https://budget-app-5y5k.onrender.com)
+</a>
+
+### Presentation Video <a name="presentation">
+[Budget App]()
+</a>
+
```

Comment:
### VIDEO PRESENTATION

- Hey, you're yet to add a video of your explaining the video, and this is a crucial part of the project requirement to have your project approved. Please, kindly add a video meeting the below requirement.

   - The maximum length of the video is 5 minutes.
   - The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
   - The video contains a demo of the project, highlighting all the project features.
   - The video contains a highlight about a piece of the code that you find interesting or proud of making.

Reply:
Done